location/src/main/java/org/motechproject/nms/location/domain/Taluka.java

```diff
@@ -0,0 +1,120 @@
+package org.motechproject.nms.location.domain;
+
+import org.motechproject.mds.annotations.Cascade;
+import org.motechproject.mds.annotations.Entity;
+import org.motechproject.mds.annotations.Field;
+import org.motechproject.mds.annotations.UIDisplayable;
+import org.motechproject.mds.domain.MdsEntity;
+
+import javax.jdo.annotations.Persistent;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity(tableName = "nms_talukas")
+public class Taluka extends MdsEntity {
+
+    @Field
+    @UIDisplayable(position = 0)
+    @NotNull
+    private String name;
+
+    @Field
+    @UIDisplayable(position = 1)
+    @NotNull
+    private Long code;
+
+    @Field
+    @UIDisplayable(position = 2)
+    @NotNull
+    private District district;
+
+    @Field
+    @Cascade(delete = true)
+    @Persistent(mappedBy = "taluka", defaultFetchGroup = "true")
+    private List<HealthBlock> healthBlocks;
```

Comment:
We thought the simpler way of working with lists (i.e. .get() as opposed to converting to an array and then grabbing elements) was worth the tradeoff.  If there was something were we need to compare equality across the elements then set probably makes sense.  In this case we aren't comparing them in that way.